clients/geth/specular/sbin/init.sh

```diff
@@ -6,3 +6,7 @@ cd $DATA_DIR
 $GETH_SPECULAR_DIR/build/bin/geth --datadir ./data_sequencer --networkid 13527 init ./genesis.json
 $GETH_SPECULAR_DIR/build/bin/geth --datadir ./data_validator --networkid 13527 init ./genesis.json
 $GETH_SPECULAR_DIR/build/bin/geth --datadir ./data_indexer --networkid 13527 init ./genesis.json
+
+cd $SBIN
+./init_clef.exp
```

Comment:
lets guard this with USE_CLEF as well (for this PR at least should preserve the legacy behavior by default)